index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Winston Churchill Page</title>
+  <link rel="stylesheet" type="text/css" href="css/style.css">
+
+</head>
+
+<body>
+
+
+<h1> History will be <span.bold>kind</span.bold> to me </h1>
+  <h1>for I intend to <b>write</b> it. 
```

Comment:
:hammer: Tidy up the indentation in this file. Also, we don't use the `<b>` tags anymore, even though they technically still work. If you want something bold, you can wrap that text in a span and style it with css like so:
HTML:
```html
<p>
  Only the word <span class="bold">bold</span> will be <span class="bold">bold</span>.
</p>
```

```css
.bold {
  font-weight: bold;
}
```